sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -84,18 +84,24 @@ def _get_attributes(
             )
         return None
 
-    def get_cryptography_client(self, key_name: str, **kwargs) -> CryptographyClient:
-        """Gets a :class:`~azure.keyvault.keys.crypto.CryptographyClient` for the given key.
+    def get_cryptography_client(
+            self,
+            key_name: str,
+            *,
+            key_version: "Optional[str]" = None,
```

Comment:
Removing kwargs is a breaking change. Is this by design?

Reply:
`kwargs` isn't being passed along to the `CryptographyClient` constructor and was only used to pass in `key_version`, so any additional kwargs wouldn't have affected the resulting client. This method returns a client that can only perform local operations, so the other client-configuring kwargs wouldn't have an effect since we're not contacting the service.

It does technically mean that if someone had been passing along extra kwargs, even though they didn't do anything and no other kwargs were documented, this method call would now be broken -- so maybe we should leave in `**kwargs`? What do you think?

Reply:
We discussed this awhile back and I think we said that we at least need a minor version bump to remove `**kwargs`. Personally, I'd leave it in, but curious what @xiangyan99 and @kashifkhan think :)

Reply:
safer to leave it in imo, I would assume `key_version` is heavily used in KV in general. Either way in the future if you want to pass through kwargs elsewhere you have it there

Reply:
Agreed. Let's just have it there.